.eslintignore

```diff
@@ -1,5 +1,7 @@
 node_modules/*
 vendor/*
+project/*
+p_*/*
```

Comment:
Where I remember project but where did `p_*/*` come from?

Reply:
that's particular to my setup, I use `p_projectname` to swap project directories. everything is returning errors its very annoying.

I'll remove that line

Reply:
You can add personal ignores to `.git/info/exclude` for yourself.